openapi.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.1
 info:
-  version: 4.170.0
+  version: 4.171.0
```

Comment:
```suggestion
  version: 4.170.2
```
This has to match the API version right?

Reply:
We're going to open a new PR with the right spec number